### PR TITLE
Add listtags feature

### PR DESCRIPTION
**Pull request by SherChew27**

### CODE REVIEW

**Comment by zeonchew0324** [comment]
LGTM

File: src/test/java/seedu/address/logic/commands/TagListCommandTest.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import org.junit.jupiter.api.Test;
+
+import seedu.address.model.Model;
+import seedu.address.model.ModelManager;
+import seedu.address.model.UserPrefs;
+
+/**
+ * Contains tests for TagListCommand.
+ */
+public class TagListCommandTest {
```
Comment (jiaxinnns) [inline src/test/java/seedu/address/logic/commands/TagListCommandTest.java:16]: Can consider adding test cases for invalid inputs like `taglist test`, `taglist 123` etc.

**Review by jiaxinnns — APPROVED** [review]
LGTM, the test cases could handle more edge cases
